src/utils/polygon.h

```diff
@@ -449,15 +449,19 @@ class PolygonRef : public ConstPolygonRef
     /*! 
      * Removes consecutive line segments with same orientation and changes this polygon.
      * 
-     * Removes verts which are connected to line segments which are both too small.
-     * Removes verts which detour from a direct line from the previous and next vert by a too small amount.
+     * 1. Removes verts which are connected to line segments which are too small.
+     * 2. Removes verts which detour from a direct line from the previous and next vert by a too small amount.
+     * 3. Moves a vert when a small line segment is connected to a much longer one. in order to maintain the outline of the object.
```

Comment:
I think you should call it 'merging the 2 verts' rather than 'moving one vert'.

Reply:
It's not _quite_ merging them.  I would interpret merging as averaging the two points.

Reply:
Yeah, but it's even less moving one point. Maybe just 'dissolving the middle segment'?